packages/calcite-components/src/components/stepper/stepper.tsx

```diff
@@ -401,18 +386,18 @@ export class Stepper implements LocalizedComponent, T9nComponent {
   }
 
   private determineActiveStepper(): void {
-    const totalItems = this.items.length;
-    if (!this.elWidth || !totalItems || this.layout !== "horizontal-single" || totalItems === 1) {
+    const { items } = this;
+
+    if (items.length < 2) {
       return;
     }
-    const activePosition = this.currentActivePosition || 0;
 
-    if (this.layout === "horizontal-single") {
-      this.multipleViewMode = false;
-      this.items.forEach((item: HTMLCalciteStepperItemElement, index) => {
-        item.hidden = index !== activePosition;
-      });
-    }
+    const { currentActivePosition, layout } = this;
+
+    this.multipleViewMode = layout !== "horizontal-single";
+    items.forEach((item, index) => {
+      item.hidden = layout === "horizontal-single" && index !== (currentActivePosition || 0);
```

Comment:
Can we use an internal property instead of `hidden`? This would overwrite any user defined `hidden` attributes/properties. Feel free to defer to another issue :)

Reply:
I totally thought of https://github.com/Esri/calcite-design-system/issues/8623 when I updated this. 🧠 It covers `stepper` already, so will defer to that issue.